lib/Remote/Payload.php

```diff
@@ -119,7 +119,7 @@ public function mergePhpConfig(array $phpConfig): void
 
     public function setPhpPath($phpPath): void
     {
-        $this->phpPath = escapeshellarg($phpPath);
+        $this->phpPath = $phpPath ? escapeshellarg($phpPath) : '';
```

Comment:
is this a phpstan issue or php 8.1 ? should we throw an exception if the string is empty?

Reply:
I didn't want to add types to the signature as that would be an API change.  That is probably the better solution in general, but I was going for the absolute minimum impact to make the errors go away as I didn't have the broader context.

Thanks for the merge. I look forward to a release so that I can benchmark a new 8.1 library I'm working on. :smile:

Reply:
ideally I want to get GH actions running with 8.1 before tagging, but Xdebug isn't behaving for some reason, need to investigate...

Reply:
https://github.com/phpbench/phpbench/releases/tag/1.1.2

the tests worked locally with 8.1+ xdebug, so disabled them in 8.1/GH actions for now